okdata/sdk/dataset_authorizer/simple_dataset_authorizer_client.py

```diff
@@ -9,50 +7,59 @@ def __init__(self, config=None, auth=None, env=None):
         super().__init__(config, auth, env)
         self.dataset_authorizer_url = self.config.get("simpleDatasetAuthorizerUrl")
 
-    def create_dataset_access(self, dataset_id, prinical_id, bearer_token=None):
+    def create_dataset_access(
+        self, dataset_id, prinical_id, bearer_token=None, retries=0
+    ):
         create_dataset_access_url = f"{self.dataset_authorizer_url}/{dataset_id}"
         request_data = {"principalId": prinical_id}
         if bearer_token:
-            result = requests.post(
+            result = self.prepared_request_with_retries(retries).post(
```

Comment:
Og denne?

Reply:
Her er det oppgitt en verdi for `bearer_token`, situasjoner der det er tilfellet er f.eks hvis en tjeneste som metadata-api skal gjøre dette kallet på vegne av en bruker. Hvis det da hadde blitt brukt self.post istedet så ville et token som representerer tjenesten blitt brukt i authorzation-headeren istedenfor tokenet som kommer inn som argument